website/docs/ephemeralstoragelimit.md

```diff
@@ -1,374 +0,0 @@
----
```

Comment:
curious, what was the issue? this was added to wrong path manually?

Reply:
Yeah. That's my guess. It's hard to validate it using commit history. But someone unsure how we generate docs is likely to do it.

Reply:
@nilekhc - Agreed, I wasn't able to replicate the issue and don't see how this could have resulted in `docs` directly? https://github.com/open-policy-agent/gatekeeper-library/blob/master/scripts/website/generate.go#L152

Reply:
Thanks for validating @apeabody 😄